LucidTests/Stores/RecoverableStoreTests.swift

```diff
@@ -45,22 +45,24 @@ final class RecoverableStoreTests: StoreTests {
         ).storing
     }
 
-    override func tearDown() async throws {
-        let success = await StubCoreDataManagerFactory.shared.clearDatabase()
-        if success == false {
-            XCTFail("Did not clear database successfully.")
-        }
+    override func asyncTearDown(_ completion: @escaping () -> Void) {
```

Comment:
Why was this change required? I'm not fully understanding the difference but the previous version using the `override func tearDown() async throws` was working as expected. Any particular case why we need the completion one?

Reply:
This might have just been a collision between our changes. I had written my code using that function. I'm not entirely sure it's necessary TBH, but I was too deep into the code to step back and evaluate.